src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -0,0 +1,4621 @@
+      "required": [
+        "transactionDate"
+      ],
+      "properties": {
+        "allocations": {
+          "type": "array",
+          "description": "List of the allocations composing this segment",
+          "items": {
+            "$ref": "#/definitions/Allocation"
+          }
+        },
+        "approvedAmount": {
+          "description": "The approved amount of the expense entry",
+          "$ref": "#/definitions/Amount"
+        },
+        "budgetAccrualDate": {
+          "type": "string",
+          "example": "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'"
+        },
+        "businessPurpose": {
+          "type": "string",
+          "description": "The purpose of the Request entry"
+        },
+        "comments": {
+          "description": "Historical comments to the Expense",
+          "$ref": "#/definitions/ResourceLink"
+        },
+        "custom1": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom10": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom11": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom12": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom13": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom14": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom15": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom16": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom17": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom18": {
+          "description": "The details from the Custom fields. These fields may not have data, depending on the configuration",
+          "$ref": "#/definitions/CustomField"
+        },
+        "custom19": {
+         
```

Comment:
The code of the Segment Type. This will most frequently be a String, but could also be an integer in the case of a custom Expense Type. For example, would be REQ_SEG_AIRFR / AIRFR for a regular air segment, but REQ_SEG_AIRFR / 10325 for a custom air segment.